src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/entity/VideoAppointment.kt

```diff
@@ -35,11 +35,19 @@ data class VideoAppointment(
 
   val appointmentType: String,
 
-  val prisonLocationId: UUID,
+  override val prisonLocationId: UUID,
 
   val appointmentDate: LocalDate,
 
-  val startTime: LocalTime,
+  override val startTime: LocalTime,
 
-  val endTime: LocalTime,
-)
+  override val endTime: LocalTime,
+) : AppointmentSlot
+
+interface AppointmentSlot {
```

Comment:
Common interface.  I might move this in following PRs but this will work as a first cut.